server/lib/host-metrics.js

```diff
@@ -222,38 +222,37 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
     },
   );
 
-  let legacyResults = [];
-  const newHostFeeIntroductionDate = new Date('2021-01-01T00:00:00.000Z');
-  if (startDate < newHostFeeIntroductionDate) {
```

Comment:
It looks to me like it was working:

```es6
{
  startDate: 2021-01-01T00:00:00.000Z,
  newHostFeeIntroductionDate: 2021-01-01T00:00:00.000Z,
  'startDate < newHostFeeIntroductionDate': false
}
{
  startDate: 2020-01-01T00:00:00.000Z,
  newHostFeeIntroductionDate: 2021-01-01T00:00:00.000Z,
  'startDate < newHostFeeIntroductionDate': true
}
```

I see that as a cheap optimization to not fetch data when we don't have to.

Reply:
Let me double check if we can live with it.

Reply:
Ok, the date is wrong, because in the case of OSC, the legacyResults are before june, not before january. If we want to keep it, I would put July 1st as date (mid-year, to simplify).